frontend/src/components/Footer/index.jsx

```diff
@@ -1,13 +1,49 @@
 import SFooter from "./style";
+import Modal from "../Modal";
 
 export default function Footer() {
   return (
     <SFooter>
-      <ul>
-        <li>About</li>
-        <li>Legal</li>
-        <li>The team</li>
-      </ul>
+      <Modal button="About">
+        <h2>Hello Modal </h2>
+        <p>
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Doloribus,
+          itaque fuga? Maxime corporis, nemo ipsum sunt porro veritatis soluta
+          illo, aspernatur eligendi ea doloremque velit natus totam? Nihil,
+          facere quae sint porro soluta corporis veritatis tempora obcaecati?
+          Aspernatur, officiis adipisci!
```

Comment:
Have you asked the client for the final texts ? If that's not the case, he won't give any and you'll stay stuck with filler text

Reply:
Yes we asked, he told us to put some lorem ipsum waiting for his text.